### DISCUSSION TITLE
Error encountered while transcribing low resource indian languages - Malayalam and Assamese

### DISCUSSION THREAD

**Discussion by shubzk** [openai/whisper#1791] [Q&A] [answered]
While transcribing Malayalam and Assamese audio by using the large-v3 model, the transcription generated is in Punjabi. Help me with a solution if you have encountered the same with any other languages.
![WhisperLanguageError](https://github.com/openai/whisper/assets/144994604/62f9cff2-6324-4f5a-a891-d06089918d4c)

**Comment by jongwook** [comment, marked as answer]
I'm afraid Whisper is not practically usable for Malayalam or Assamese. Similar discussions regarding Amharic in:

- #1735

**Comment by nairajay2k** [comment]
Any roadmap to fix for malayalam etc?
